src/runtime/mod.rs

```diff
@@ -243,14 +243,13 @@ impl Module {
         std::fs::create_dir_all(&log_dir)?;
         // Open a file for appending. Right now this will just keep appending as there is no log
         // rotation or cleanup
-        let stderr = unsafe {
-            cap_std::fs::File::from_std(
-                std::fs::OpenOptions::new()
-                    .append(true)
-                    .create(true)
-                    .open(log_dir.join(STDERR_FILE))?,
-            )
-        };
+        let stderr = cap_std::fs::File::from_std(
+            std::fs::OpenOptions::new()
+                .append(true)
+                .create(true)
+                .open(log_dir.join(STDERR_FILE))?,
+            ambient_authority(),
```

Comment:
`ambient_authority()` follows the way the Wasmtime CLI is currently set up to work -- https://github.com/bytecodealliance/wasmtime/blob/f958c014441be5534f2a3eaeb3bbbf180257bb79/src/commands/run.rs#L230

See https://github.com/sunfishcode/ambient-authority.